README.md

```diff
@@ -1,8 +1,6 @@
 # Escalate
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/escalate`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+A simple and lightweight gem to help with escalating errors
```

Comment:
A simple and lightweight gem to escalate rescued exceptions.

Reply:
I think we should mention somewhere up front here that it defines an abstract interface where concrete implementations can easily be written. For example:
- log to gem's logger (default)
- log to `STDERR` if no logger found
- Honeybadger
- support for other error escalation services like Airbrake and Sentry coming soon